CMakeLists.txt

```diff
@@ -908,7 +908,7 @@ if (ENABLE_EXAMPLES)
 endif()
 
 
-if (ENABLE_UNITTESTS AND ENABLE_CXX11)
+if (ENABLE_UNITTESTS)
```

Comment:
Unit tests should be enabled only with C++11 simply because the test driver library requires it - it doesn't matter if it's required by the test procedure itself.

Reply:
With this change GCC 4.6.3 which is not C++11 can generate a working test_srt.
Here the cmake command I use `cmake . -DUSE_ENCLIB=openssl -DENABLE_CXX11=OFF -DENABLE_UNITTESTS=1 -DHAVE_COMPILER_GNU_COMPAT=OFF -DINSTALL_GTEST=OFF`

Reply:
But I guess if you set `-DINSTALL_GTEST=OFF`, you must have had the gtest library already installed and compiled by a C++11-compliant compiler.

And anyway, `ENABLE_UNITTESTS` enables all unit tests, and it suffices that at least one of them requires C++11, as most of them do, to require C++11-compliance for all of them. The idea for the unit tests is that they are run automatically, so even splitting them into those that require C++11 and those that don't doesn't make much sense, as all are intended to be run anyway.